torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -125,10 +133,33 @@ def _retrieve_or_adapt_input_to_graph_set(fx_node_arg, fx_name_to_onnxscipt_valu
         # 2. fx_node_arg (variable in torch.fx.Graph) is be mapped to
         #    torch.jit.Value, fx_name_to_onnxscipt_value[fx_node_arg.name],
         #    in TorchScript graph.
-        onnx_tensor = fx_name_to_onnxscipt_value[onnx_tensor.name]
-    elif isinstance(onnx_tensor, torch.dtype):
+        return fx_name_to_onnxscipt_value[onnx_tensor.name]
+    if isinstance(onnx_tensor, (tuple, list)) and any(
+        isinstance(ele, torch.fx.Node) for ele in onnx_tensor
+    ):
+        # This intends to solve dynamic axes. for example, if the size input of op.Expand
+        # is dynamic, each of dim would be originated from other tensors.
+        sequence_elements = []
+        for tensor in onnx_tensor:
+            if isinstance(tensor, torch.fx.Node):
+                sequence_elements.append(fx_name_to_onnxscipt_value[tensor.name])
+            else:
+                sequence_elements.append(tensor)
+        # Concat all the elements in the sequence.
+
+        # For example:
+        #    inputs: [2, 4, TensorA, 1, TensorB]
+        #    outputs: Tensor([Tensor(2), Tensor(4), TensorA, Tensor(1), TensorB])
+
+        # onnx-script auto wrap constants with op.Constants,
```

Comment:
Humm, I don't see the key explanation in the doc. Please say `shapes are mapped to tensors in ONNX graph (type: TorchScriptGraph?), so list of sym_ints is concatenated to a tensor before calling ONNX op.`

Reply:
I was thinking this is not only applied to SymInt, it's for all List[fx.Node] scenario. If I update the comment to specify SymInt, I think we need to add another elif branch for those list of fx.Node without SymInt value?

Reply:
I don't have a good idea to handle `all` cases but just having a specific branch for sym_* sounds good enough.

Reply:
Done